components/builder-web/app/shared/statuspage/statuspage-indicator.component.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Input, AfterViewInit, Component } from '@angular/core';
+
+@Component({
+  selector: 'hab-statuspage',
+  template: `
+    <a href="//status.habitat.sh/" target="_blank">
+      <span class="color-dot {{indicator}}"></span>
+      <span class="color-description">{{description}}</span>
+    </a>
+  `
+})
+export class StatuspageIndicatorComponent implements AfterViewInit {
+  @Input() indicator;
```

Comment:
It doesn't look like this needs to take inputs; we can probably just declare these as local properties.